pages/linux/systemctl.md

```diff
@@ -3,9 +3,10 @@
 > Control the systemd system and service manager.
 > More information: <https://www.freedesktop.org/software/systemd/man/systemctl.html>.
 
-- Show all running services:
+- List all services (optionally filtering by state):
 
-`systemctl status`
+`systemctl list-unit-files --type=service --all`
```

Comment:
Probably this is a typo IMO.
Can you please remove one of the lines here which is not appropriate